src/Hyperion/Extensions/TypeEx.cs

```diff
@@ -132,19 +144,82 @@ private static Type GetTypeFromManifestName(Stream stream, DeserializerSession s
             });
         }
 
+        public static bool disallowUnsafeTypes = true;
+
+        private static ReadOnlyCollection<string> unsafeTypesDenySet =
```

Comment:
This list was amalgamated from:

 - Messagepack-csharp's [type exclusion set](https://github.com/neuecc/MessagePack-CSharp/blob/4142e346ead283997c46f7257f36a3037177bccf/src/MessagePack/Formatters/TypelessFormatter.cs#L97)
 - ysoserial.net's [set of gadgets](https://github.com/pwntester/ysoserial.net)